layouts/index.html

```diff
@@ -205,16 +205,16 @@ <h3 class="mt-6 text-white">Install with Conda</h3>
                     <br>
                     <p><strong>2.</strong> Then quick install RAPIDS with:</p>
                     <pre
-                        class="highlight use-white-copy"><code>conda create -n rapids-25.04 -c rapidsai -c conda-forge -c nvidia rapids=25.04 python=3.12 cuda-version=12.8</code></pre>
+                        class="highlight use-white-copy"><code>conda create -n rapids-25.06 -c rapidsai -c conda-forge -c nvidia rapids=25.06 python=3.12 cuda-version=12.8</code></pre>
```

Comment:
We can now recommend users to Python 3.13 and CUDA 12.9 as defaults for runtime environments.
```suggestion
                        class="highlight use-white-copy"><code>conda create -n rapids-25.06 -c rapidsai -c conda-forge -c nvidia rapids=25.06 python=3.13 cuda-version=12.9</code></pre>
```